package.json

```diff
@@ -42,8 +42,8 @@
     "check": "tsc --project ./tsconfig.check.json && pnpm lint",
     "postinstall": "pnpm i rimraf && pnpm build",
     "prepare": "husky install",
-    "generate-types-from-schema": "cd src/export/digitalPlanning/schema/ && json2ts schema.json > types.d.ts",
-    "copy-json-schema": "copyfiles -f ./src/export/digitalPlanning/schema/schema.json ./dist/export/digitalPlanning/schema"
+    "generate-types-from-schema": "cd src/export/digitalPlanning/schemas/ && json2ts application/schema.json > application/types.d.ts && json2ts preApplication/schema.json > preApplication/types.d.ts",
+    "copy-json-schema": "copyfiles -f ./src/export/digitalPlanning/schemas/application/schema.json ./dist/export/digitalPlanning/application/schema && copyfiles -f ./src/export/digitalPlanning/schemas/preApplication.json ./dist/export/digitalPlanning/preApplication/schema"
```

Comment:
These commands aren't the most eloquent and will be worth refactoring to crawl the directory or similar if we find ourselves with a third schema, but they work for now (and no known plans for a third schema anytime soon)!